jdbc/src/main/java/nextstep/jdbc/RowMapperResultSetExtractor.java

```diff
@@ -0,0 +1,39 @@
+package nextstep.jdbc;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+public class RowMapperResultSetExtractor<T> {
+
+    private final RowMapper<T> rowMapper;
+    private final int rowsExpected;
+
+    public RowMapperResultSetExtractor(RowMapper<T> rowMapper) {
+        this(rowMapper, 0);
+    }
+
+    public RowMapperResultSetExtractor(RowMapper<T> rowMapper, int rowsExpected) {
+        if (Objects.isNull(rowMapper))
+            throw new IllegalStateException("RowMapper is required");
```

Comment:
```suggestion
        if (Objects.isNull(rowMapper)) {
            throw new IllegalStateException("RowMapper is required");
        }
```

중괄호 누락으로 인한 버그가능성은 생각보다 큽니다! 꼭 작성해주세요 :)